src/test/resources/data/FeedbackSessionResultsTest.json

```diff
@@ -594,7 +594,7 @@
       "feedbackSessionName": "Standard feedback session",
       "courseId": "FSRTest.idOfTypicalCourse1",
       "feedbackQuestionId": "7",
-      "giver": "FSRTest.student2InCourse1@gmail.tmt",
+      "giver": "Team 1.1",
```

Comment:
And therefore, let's make sure we have test data that cover the legacy data. We should remove it only after we have migrated the database.

Reply:
How does this get addresed?

Reply:
Changes have been reverted